zad4/app/controllers/team_controller.rb

```diff
@@ -0,0 +1,21 @@
+class TeamController < ApplicationController
+
+  include TeamHelper
+
+  def show
+    players = Array.new
+    captains = Array.new
+    
+    Player.all.shuffle.each do |player|
```

Comment:
To jest spoko, rubiowe podejście, ale zazwyczaj bardziej cenione jest podejście od strony bazy danych :) NP. można było to załatwić w tym stylu i przy pomocy sztuczki której nie było na wykłądzie, ale wujek google wie :D :
```ruby
players = Player.order('random()').where(captain: false).first(20)
```
Analogicznie kapitanowie :)